### ISSUE TITLE
/usr/sbin/biosdevname --policy=all_ethN -i bash: line 3: /usr/sbin/biosdevname: No such file or directory

### ISSUE THREAD

**Issue by holms** [holms/vagrant-centos7-box#1]
When setting private_ip, this box fails to load. Seems to be this one is related to mitchellh/vagrant#2614

Seems to be that vagrant expects eth0 as network interface. They even been writing patch for this.. although it doesn't work for me. 

```
==> dev: Machine booted and ready!
==> dev: Checking for guest additions in VM...
==> dev: Configuring and enabling network interfaces...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

/usr/sbin/biosdevname --policy=all_ethN -i bash: line 3: /usr/sbin/biosdevname: No such file or directory

Stdout from the command:



Stderr from the command:

bash: line 3: /usr/sbin/biosdevname: No such file or directory

```

**Comment by du5tball** [comment]
Same problem here, can be fixed by installing biosdevname from the repos.

**Comment by holms** [comment]
@du5tball Check next commit, which fixes this problem, but it will come with next problem

```
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

/sbin/ifup eth2

Stdout from the command:

ERROR    : [/etc/sysconfig/network-scripts/ifup-eth] Error, some other host already uses address 10.1.2.5.


Stderr from the command:

```

Seems to be setting `config.vm.network "private_network", ip: "10.1.1.5", :adapter => 2` works :) You have to specify adapter number, first adapter is always NAT.

**Comment by du5tball** [comment]
Actually, vagrant expects the biosdevname binary to exist. I built a machine with the original base.sh, but added biosdevname to ks.cfg. Lo and behold, everything works as expected, no further errors.

**Comment by holms** [comment]
@du5tball well for me not. I had to set grub parameters, and then removed one of udev configs. Then that error disappeared. All other combinations failed for me, been playing with this crap for 20 hours.

After that's been solved, I've got another issue with that `ifup eth2` error. I'm using private_network ip address. When I've started to specify adapter, that error went away :) When you're not using private_network configuration everything works. But I need private network configuration, because I need to test cluster setup locally, with some highly confidential data :)
